### PR TITLE
BLISS Section 1.1 on Research Question

### PR DESCRIPTION
**Pull request by SylLutze**

### CODE REVIEW

File: docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst [hunk 1]
```diff
@@ -100,10 +100,137 @@ audience.
...
+ethnicity, and/or ancestry are relevant, and, if so, what the casual pathways might be. Spending 
+time considering the causal pathways will help you consider what exactly you are 
+investigating and might identify more relevant proxy measures.
+
+To work through this example, let’s consider a starting research question: how will 
+the rates of cardiovascular disease (CVD) change over the next 5 years in the US?
+
+First, let’s consider if including race, ethnicity, and/or ancestry is relevant to 
+the research question. It’s commonly known that CVD rates vary with race, and that 
+race demographics are changing within the US. Understanding how different groups’
```
Comment (zmbc) [inline docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst:113]:
```suggestion
racial demographics are changing within the US. Understanding how different groups’ 
```

Not confident in this, I just feel like I've more often heard racial.

File: docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst [hunk 2]
```diff
@@ -100,10 +100,137 @@ audience.
...
+the future. So, it seems relevant to include race here.  
+
+Second, let’s think about if this will lead to changes in policies. Here again, 
+it seems plausible that if we see a sharper increase  in CVD incidence among some 
+racial groups, decision-makers might deploy racially aware intervention strategies. 
+So here, including something about race, ethnicity, and/or ancestry seems relevant. 
+
+Last, let’s think through the possible causal pathways and decide what metric(s) 
+we want to include. In the below diagrams, the solid lines indicate casual pathways, 
+while the dotted lines indicate correlation.
```
Comment (zmbc) [inline docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst:124]: The dotted arrows are something else -- I know I suggested them 😅 but maybe we should use color or something else to indicate "we don't like this arrow/it doesn't have to be this way"

File: docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst [hunk 3]
```diff
@@ -100,10 +100,137 @@ audience.
...
+.. image:: race_a_v1.svg
+
+We stated above that its commonly known that CVD rates vary with race. This is 
+well documented and true. It's easy therefore to think about the casual diagram 
+as a single line between race and disease. But all we know is that there is a 
+correlation between race and CVD. We want to more closely examine what might be 
+causing this correlation. 
+
+Given our defintion of race as physical characteristics, for race to cause the 
+disease, it means that the physical characteristics are causing the disease itself.
```
Comment (zmbc) [inline docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst:137]:
```suggestion
Race is related to physical characteristics.
We might naively assume that the physical characteristics are causing the disease itself. 
```

I don't love saying race = physical characteristics, when they have separate bubbles in this diagram. Suggested simplification.

File: docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst [hunk 4]
```diff
@@ -100,10 +100,137 @@ audience.
...
+well documented and true. It's easy therefore to think about the casual diagram 
+as a single line between race and disease. But all we know is that there is a 
+correlation between race and CVD. We want to more closely examine what might be 
+causing this correlation. 
+
+Given our defintion of race as physical characteristics, for race to cause the 
+disease, it means that the physical characteristics are causing the disease itself. 
+There might be a few cases where this causal pathway is correct, but they are limited 
+in scope. For example, lighter skinned individuals tend to get skin cancer at higher 
+rates. In this case, the physical characteristic of race impacts the disease. However,
```
Comment (zmbc) [inline docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst:140]:
```suggestion
rates. In this case, the physical characteristic which leads to racial categorization impacts the disease. However, 
```

Maybe too wordy?

File: docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst [hunk 5]
```diff
@@ -100,10 +100,137 @@ audience.
...
+
+Last, let’s think through the possible causal pathways and decide what metric(s) 
+we want to include. In the below diagrams, the solid lines indicate casual pathways, 
+while the dotted lines indicate correlation. 
+
+**Option 1a: Race** 
+
+.. image:: race_a_v1.svg
+
+We stated above that its commonly known that CVD rates vary with race. This is
```
Comment (pletale) [inline docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst:130]:
```suggestion
We stated above that it's commonly known that CVD rates vary with race. This is 
```

File: docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst [hunk 6]
```diff
@@ -100,10 +100,137 @@ audience.
...
+This casual pathway is much more common, and seems more applicable to our example with CVD. 
+However, considering the exact pathway here is important. We list a few possible pathways for 
+CVD based on race. Note that these do overlap with ethnicity in some cases. This is 
+because discrimination can impact culture/ethnic practices. These are often 
+rooted in racism. 
+
+- Historical red lining leading to African Americans living in food deserts, having less available green space, or living in neighborhoods with fewer social services
+- Historical racism leading to lower wealth in some communities, reduced access to health insurance, or limited access to quality medical care
+- Microaggressions experienced by non-white individuals leading to elevated blood pressure 
+- Racism in healthcare settings leading to delayed treatment once a person seeks care
```
Comment (zmbc) [inline docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst:164]: I think these are really helpful examples! I like how they span a wide range of time and scale.

File: docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst [hunk 7]
```diff
@@ -100,10 +100,137 @@ audience.
...
+- Historical red lining leading to African Americans living in food deserts, having less available green space, or living in neighborhoods with fewer social services
+- Historical racism leading to lower wealth in some communities, reduced access to health insurance, or limited access to quality medical care
+- Microaggressions experienced by non-white individuals leading to elevated blood pressure 
+- Racism in healthcare settings leading to delayed treatment once a person seeks care 
+
+**Option 2a: Ethnicity** 
+
+.. image:: ethnicity_a_v1.svg
+
+The second causal pathway is between ethnicity and disease. Here, there are
```
Comment (zmbc) [inline docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst:170]:
```suggestion
A second correlation often observed is between ethnicity and disease. Here, there are 
```

Maybe?

File: docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst [hunk 8]
```diff
@@ -100,10 +100,137 @@ audience.
...
+- Historical racism leading to lower wealth in some communities, reduced access to health insurance, or limited access to quality medical care
+- Microaggressions experienced by non-white individuals leading to elevated blood pressure 
+- Racism in healthcare settings leading to delayed treatment once a person seeks care 
+
+**Option 2a: Ethnicity** 
+
+.. image:: ethnicity_a_v1.svg
+
+The second causal pathway is between ethnicity and disease. Here, there are 
+some compelling arguments for CVD. For example, the mediterranean diet has
```
Comment (zmbc) [inline docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst:171]:
```suggestion
some compelling arguments that this relationship could be causal for CVD. For example, the Mediterranean diet has 
```

File: docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst [hunk 9]
```diff
@@ -100,10 +100,137 @@ audience.
...
+CVD rates are changing differently might lead us to make a better forecast for 
+the future. So, it seems relevant to include race here.  
+
+Second, let’s think about if this will lead to changes in policies. Here again, 
+it seems plausible that if we see a sharper increase  in CVD incidence among some 
+racial groups, decision-makers might deploy racially aware intervention strategies. 
+So here, including something about race, ethnicity, and/or ancestry seems relevant. 
+
+Last, let’s think through the possible causal pathways and decide what metric(s) 
+we want to include. In the below diagrams, the solid lines indicate casual pathways,
```
Comment (NathanielBlairStahn) [inline docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst:123]:
```suggestion
we want to include. In the below diagrams, the solid arrows indicate casual pathways, 
```
Would this be more accurate?

File: docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst [hunk 10]
```diff
@@ -100,10 +100,137 @@ audience.
 1.1 Research question
 ---------------------
 
-.. todo:: 
+When you’re designing a research question, it’s important to consider both if race, 
+ethnicity, and/or ancestry are relevant, and, if so, what the casual pathways might be. Spending 
+time considering the causal pathways will help you consider what exactly you are 
+investigating and might identify more relevant proxy measures.
+
+To work through this example, let’s consider a starting research question: how will
```
Comment (pletale) [inline docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst:108]: I like that CVD is our recurring example throughout this section, it provides clarity/consistency. It could also be helpful to offer an example research question where race/ethnicity/ancestry would not be as relevant to a research question?

File: docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst [hunk 11]
```diff
@@ -100,10 +100,137 @@ audience.
...
+
+.. image:: ethnicity_a_v1.svg
+
+The second causal pathway is between ethnicity and disease. Here, there are 
+some compelling arguments for CVD. For example, the mediterranean diet has 
+been shown to lower risk of cardiovascular disease. So ethnic groups that 
+tend to follow this type of diet might have lowered risk of CVD. 
+
+However, a big risk of this causal pathway is that ethnicity can be 
+greatly affected by circumstance and history.
```
Comment (zmbc) [inline docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst:176]:
```suggestion
However, this is not the only potential explanation for correlation with ethnicity,
which can be 
greatly affected by circumstance and history. 
```

I think spelling out the "risk" more explicitly helps clarify here.

File: docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst [hunk 12]
```diff
@@ -100,10 +100,137 @@ audience.
...
+Last, let’s think through the possible causal pathways and decide what metric(s) 
+we want to include. In the below diagrams, the solid lines indicate casual pathways, 
+while the dotted lines indicate correlation. 
+
+**Option 1a: Race** 
+
+.. image:: race_a_v1.svg
+
+We stated above that its commonly known that CVD rates vary with race. This is 
+well documented and true. It's easy therefore to think about the casual diagram
```
Comment (pletale) [inline docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst:131]:
```suggestion
well documented and true. It's easy therefore to think about the causal diagram 
```

File: docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst [hunk 13]
```diff
@@ -100,10 +100,137 @@ audience.
...
+we want to include. In the below diagrams, the solid lines indicate casual pathways, 
+while the dotted lines indicate correlation. 
+
+**Option 1a: Race** 
+
+.. image:: race_a_v1.svg
+
+We stated above that its commonly known that CVD rates vary with race. This is 
+well documented and true. It's easy therefore to think about the casual diagram 
+as a single line between race and disease. But all we know is that there is a
```
Comment (NathanielBlairStahn) [inline docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst:132]:
```suggestion
as a single arrow from race to disease. But all we know is that there is a 
```
Is this what you mean? Or did you want the arrow to go the other way? Or do you mean a single dotted correlation line?

File: docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst [hunk 14]
```diff
@@ -100,10 +100,137 @@ audience.
...
+to assimilate to “western” lifestyles, which often involve more processed and 
+fatty foods, they tend to have higher rates of CVD. Here, the outside force is changes in environment and cultural norms, 
+leading to changes in lifestyle and food choices, which in turn to lead to 
+differences in CVD rates. 
+
+Some other examples include: 
+
+- Slavery creating new African American cultural norms separate from African cultural norms 
+- Availability of green space or parks leading to increase exercise in some communities 
+- Food deserts leading to different food practices
```
Comment (zmbc) [inline docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst:194]: I feel like these might be a bit confusing as first examples, as they all kind of entangle ethnicity with race. Could we maybe start with a more cut-and-dry example like language discrimination?

**Review by pletale — APPROVED** [review]

**Review by zmbc — APPROVED** [review]
This is great! I love how visual this has turned out.

Can you make sure to replace "casual" with "causal" throughout? 😄

File: docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst [hunk 15]
```diff
@@ -100,10 +100,137 @@ audience.
...
+**Option 1a: Race** 
+
+.. image:: race_a_v1.svg
+
+We stated above that its commonly known that CVD rates vary with race. This is 
+well documented and true. It's easy therefore to think about the casual diagram 
+as a single line between race and disease. But all we know is that there is a 
+correlation between race and CVD. We want to more closely examine what might be 
+causing this correlation. 
+
```
Comment (NathanielBlairStahn) [inline docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst:135]: I think it would be helpful to start with the smaller diagram showing just the correlation between race and disease, as described in the paragraph above, then start a new section here for Option 1a, showing one possible explanation for the correlation, i.e. the diagram you have above, except I think it would then be clearer to omit the dotted line from that diagram because the common cause of "Physical Traits" is our first hypothesis of what's causing the correlation, so the dotted line is redundant with this common cause.
